app/templates/Tags/view.php

```diff
@@ -0,0 +1,77 @@
+            <?= $this->Html->link(__('New Tag'), ['action' => 'add'], ['class' => 'side-nav-item']) ?>
+        </div>
+    </aside>
+    <div class="column-responsive column-80">
+        <div class="tags view content">
+            <h3><?= h($tag->title) ?></h3>
+            <table>
+                <tr>
+                    <th><?= __('Title') ?></th>
+                    <td><?= h($tag->title) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Id') ?></th>
+                    <td><?= $this->Number->format($tag->id) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Created') ?></th>
+                    <td><?= h($tag->created) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Modified') ?></th>
+                    <td><?= h($tag->modified) ?></td>
+                </tr>
+            </table>
+            <div class="related">
+                <h4><?= __('Related Articles') ?></h4>
+                <?php if (!empty($tag->articles)) : ?>
+                <div class="table-responsive">
+                    <table>
+                        <tr>
+                            <th><?= __('Id') ?></th>
+                            <th><?= __('User Id') ?></th>
+                            <th><?= __('Title') ?></th>
+                            <th><?= __('Slug') ?></th>
+                            <th><?= __('Body') ?></th>
+                            <th><?= __('Published') ?></th>
+                            <th><?= __('Created') ?></th>
+                            <th><?= __('Modified') ?></th>
+                            <th class="actions"><?= __('Actions') ?></th>
+                        </tr>
+                        <?php foreach ($tag->articles as $articles) : ?>
+                        <tr>
+                            <td><?= h($articles->id) ?></td>
+                            <td><?= h($articles->user_id) ?></td>
+                            <td><?= h($articles->title) ?></td>
+                            <td><?= h($articles->slug) ?></td>
+                            <td><?= h($articles->body) ?></td>
+                            <td><?= h($articles->published) ?></td>
+                            <td><?= h($articles->created) ?></td>
+                            <td><?= h($articles->modified) ?></td>
+                            <td class="actions">
+                                <?= $this->Html->link(__('View'), ['controller' => 'Articles', 'action' => 'view', $articles->id]) ?>
+                                <?= $this->Html->link(__('Edit'), ['controller' => 'Articles', 'action' => 'edit', $articles->id]) ?>
+                                <?= $this->Form->postLink(__('Delete'), ['controller' => 'Articles', 'action' => 'delete', $articles->id], ['confirm' => __('Are you sure you want to delete # {0}?', $articles->id)]) ?>
```

Comment:
```suggestion
                                <?= $this->Html->link(__('View'), ['controller' => 'Articles', 'action' => 'view', $articles->slug]) ?>
                                <?= $this->Html->link(__('Edit'), ['controller' => 'Articles', 'action' => 'edit', $articles->slug]) ?>
                                <?= $this->Form->postLink(__('Delete'), ['controller' => 'Articles', 'action' => 'delete', $articles->slug], ['confirm' => __('Are you sure you want to delete # {0}?', $articles->id)]) ?>
```

The routing array for controller Articles and action view, edit, delete used slug. Refer

https://github.com/sergio9977/tutorial-cakephp/blob/45667057597f480632498c5d35551af6a24e558f/app/src/Controller/ArticlesController.php#L45

https://github.com/sergio9977/tutorial-cakephp/blob/45667057597f480632498c5d35551af6a24e558f/app/src/Controller/ArticlesController.php#L89

https://github.com/sergio9977/tutorial-cakephp/blob/45667057597f480632498c5d35551af6a24e558f/app/src/Controller/ArticlesController.php#L122

Please try access the URL `http://127.0.0.1/tags/view/{TAG_ID}`. Replace the placeholder {TAG_ID} with your records stored in table tags.

Then please access the link for Articles (either View, Edit or Delete). Then we'll get error `Record not found in table "articles"`.

The error occurred because we pass the articles' id instead of slug. Please fix and ensure it's working correctly.